_posts/2021-05-06-ion-js-4_2_1-released.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: news_item
+title: "Ion JS 4.2.0 and 4.2.1 Released"
+date: 2021-05-06
+categories: news ion-js
+---
+
+API Changes:
+
+    Added position() API to Reader
+    Added an equals() API for the DOM
+    Timestamp’s constructor can now accept a Date instead of requiring individual time unit fields.
+    Reader.byteValue is now deprecated in favor of the new alias: Reader.uInt8ArrayValue, which is more descriptive.
+    Added support for duplicate fields in Struct for the DOM.
```

Comment:
These are currently rendering as code blocks. Was that intentional, or did you want them to be bullet lists? (Asking in case it was the result of a a lossy copy/paste.)

Reply:
modified this with d6a96f7